lite-rpc/src/bridge.rs

```diff
@@ -323,9 +352,34 @@ impl LiteRpcServer for LiteBridge {
 
     async fn get_recent_prioritization_fees(
         &self,
-        _pubkey_strs: Option<Vec<String>>,
+        pubkey_strs: Vec<String>,
     ) -> crate::rpc::Result<Vec<RpcPrioritizationFee>> {
-        todo!()
+        // This method will get the latest global and account prioritization fee stats and then send the maximum p75
+        let accounts = pubkey_strs
+            .iter()
+            .filter_map(|pubkey| Pubkey::from_str(pubkey).ok())
+            .collect_vec();
+
+        let global_prio_fees = self.prio_fees_service.get_latest_priofees().await;
+        let max_p75 = global_prio_fees
+            .map(|(_, fees)| {
+                let fees = fees.get_percentile(0.75).unwrap_or_default();
+                std::cmp::max(fees.0, fees.1)
+            })
+            .unwrap_or_default();
+
+        let mut ret: Vec<RpcPrioritizationFee> = vec![];
+        for account in accounts {
+            let (slot, stats) = self.account_priofees_service.get_latest_stats(&account);
```

Comment:
I see this function uses `unwrap_or_default`. How does it work if the account does not exist ?